R/utils.R

```diff
@@ -1,10 +1,6 @@
-# TODO(#2768): possibly just use %||% instead
-`%|||%` <- function(x, y) {
-  if (is.null(x) || length(x) == 0L || (is.atomic(x[[1L]]) && is.na(x[[1L]]))) {
-    y
-  } else {
-    x
-  }
+# backports approach doesn't work since we need this before .onLoad() in names2()
+if (!exists("%||%", "package:base")) {
```

Comment:
Mark this with a TODO to delete once support for 4.1 is dropped?

Reply:
4.4, but yes, good idea